ext/vscode/package.nls.json

```diff
@@ -1,98 +1,32 @@
 {
     "azure-dev.commands_category": "Azure Developer",
 
-    "azure-dev.commands.util.terminalTitle": "az dev",
```

Comment:
All these strings that were used only in TypeScript source were redundant--`vscode-nls` _only_ uses `package.nls.json` for strings in `package.json`, not those in source.